src/main/java/com/agency04/sbss/pizza/model/Marinara.java

```diff
@@ -0,0 +1,25 @@
+package com.agency04.sbss.pizza.model;
+
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * Model of pizza representing concrete pizza
+ */
+public class Marinara implements Pizza {
+
+	@Override
+	public String getName() {
+		return "Marinara";
+	}
+
+	@Override
+	public Set<PizzaIngredient> getIngredients() {
+		Set<PizzaIngredient> ingredients = new HashSet<>();
```

Comment:
Pripazi na činjenicu da vraćaš promjenjivi set, alternativa bi bila i korištenje `Set.of()` metode, u tom slučaju vraćaš nepromjenjivi, non-nullable, thread-safe set, isto tako se možeš igrati s access modifierima i ključnim riječima (tipa `private final` uz promjenu mjesta defincije/scopea varijable).
Pohvala za `HashSet` koji je generalno dobra praksa, ali u slučaju kad barataš isključivo enumima, `EnumSet` je ipak way to go (efikasniji kod).
Od prijedloga razmotri koji želiš, nema "krivog" :)